src/views/Dashboard/Dashboard.tsx

```diff
@@ -492,16 +506,16 @@ const Dashboard: React.FC = () => {
     <Page>
       <PageHeader icon="📊" subtitle="Overview of the YAM ecosystem" title="YAM Dashboard" />
       <Container size="lg">
-        <TopCards />
+        <TopCards currentPrice={currentPrice} change24={change24} maxSupply={maxSupply} marketCap={marketCap} treasuryValue={treasuryValue} />
```

Comment:
would be ideal to combine each card to be individually separate `currentPrice={currentPrice} change24={change24}` can just be `currentPrice={currentPrice}` with the 24h change inside of it somewhere